3.0-RELEASE-NOTES.md

```diff
@@ -127,3 +127,16 @@ removed properties together with the middleware module name to use instead:
 We have also removed `loopback.mime`, which was always set to `undefined`.
 
 See [loopback#2349](https://github.com/strongloop/loopback/pull/2394).
+
+## remove `loopback.errorhandler`
```

Comment:
^^^ click on the "edit" icon of the comment above, copy the markdown source and use it to replace the content of the new section you are adding.